cluster-autoscaler/cloudprovider/aws/aws_manager.go

```diff
@@ -259,6 +259,14 @@ func (m *AwsManager) buildNodeFromTemplate(asg *asg, template *asgTemplate) (*ap
 		}
 	}
 
+	resourcesFromTags := extractAllocatableResourcesFromAsg(template.Tags)
+	if val, ok := resourcesFromTags["ephemeral-storage"]; ok {
+		node.Status.Capacity[apiv1.ResourceEphemeralStorage] = *val
+	}
+
+	// TODO: use proper allocatable!!
+	node.Status.Allocatable = node.Status.Capacity
```

Comment:
Please drop this, we already have better logic above and this would just overwrite it.